src/models/OAuthModel.php

```diff
@@ -125,20 +125,25 @@ public function createAccessTokenFromPassword($clientId, $username, $password)
      */
     protected function getUserId($username, $password)
     {
-        $sql  = 'SELECT ID, password, email FROM user
-            WHERE username=:username
-            AND verified = 1';
+        $sql  = 'SELECT ID, password, email, verified FROM user
+            WHERE username=:username';
         $stmt = $this->_db->prepare($sql);
         $stmt->execute(array("username" => $username));
```

Comment:
The if (!$result) should catch in that instance.  It's also not something that we appear to be doing in other places, tho I'm happy to put it in if it's required.